src/payment/payment-request-body.ts

```diff
@@ -4,19 +4,21 @@ import { InternalCustomer } from '../customer';
 import { InternalOrder } from '../order';
 import { InternalShippingOption } from '../shipping';
 
+import { PaymentAdditionalAction } from '.';
```

Comment:
Usually we import using the exact path instead of from index when the file is in the same directory (basically whenever you see `'.'`) i.e.: `import PaymentAdditionalAction  from './payment-additional-action'` 💅